reflex/utils/compat.py

```diff
@@ -69,3 +69,18 @@ def pydantic_v1_patch():
 
 with pydantic_v1_patch():
     import sqlmodel as sqlmodel
+
+
+def sa_column_id_supporter(field) -> bool:
+    """Adds support for sa_column so we don't add id with a primary key
+    is defined
+
+    AKA simons_litttle_helper
+    AKA masens_litttlerrr_helperr
+    """
```

Comment:
😀 docstring must follow repo conventions

Reply:
I believe i have updated to match the repo convention

Reply:
you need to run pre-commit

```
poetry run pre-commit run --all-files
```